config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "GITHUB_TOKEN": "02c488529ccb97674c4655467617f0442f37cee6",
```

Comment:
What is the best practice for hiding the access token? If we remove it, the page won't load anything.